From 65d50661ac2b27722d37896900d10f0f7545d5cd Mon Sep 17 00:00:00 2001
From: bitrvmpd <e.noyer.silva@gmail.com>
Date: Fri, 5 Jun 2020 12:15:57 -0500
Subject: [PATCH] :bug: Added missing Kconfig flags for prima/pronto

---
 drivers/staging/prima-latest/Kconfig | 19 +++++++++++++++++++
 1 file changed, 19 insertions(+)

diff --git a/drivers/staging/prima-latest/Kconfig b/drivers/staging/prima-latest/Kconfig
index 63894afc14e4..715ced910775 100644
--- a/drivers/staging/prima-latest/Kconfig
+++ b/drivers/staging/prima-latest/Kconfig
@@ -29,6 +29,18 @@ config PRIMA_WLAN_LFR
 	bool "Enable the Prima WLAN Legacy Fast Roaming feature"
 	default n
 
+config WLAN_FEATURE_SAE
+	bool "Enable SAE"
+	default n
+
+config WLAN_AKM_SUITE_OWE
+	bool "Enable OWE"
+	default n
+
+config PRIMA_WLAN_LFR_MBB
+	bool "Enable Legacy Fast Roaming(LFR) Make Before Break"
+	default n
+
 config PRIMA_WLAN_OKC
 	bool "Enable the Prima WLAN Opportunistic Key Caching feature"
 	default n
@@ -37,6 +49,9 @@ config PRIMA_WLAN_11AC_HIGH_TP
 	bool "Enable the Prima WLAN 802.11ac High Throughput option (depends upon kernel support)"
 	default n
 
+config MDNS_OFFLOAD_SUPPORT
+	bool "Enable mDNS feature"
+
 config WLAN_FEATURE_11W
 	bool "Enable the WLAN 802.11w Protected Management Frames feature"
 	default n
@@ -57,4 +72,8 @@ config QCOM_TDLS
 	bool "Enable TDLS"
 	default n
 
+config WLAN_FEATURE_AP_FIND
+	bool "Enable AP Find feature"
+	default n
+
 endif # PRIMA_WLAN || PRONTO_WLAN
-- 
2.25.1

